params/config.libevm.go

```diff
@@ -108,7 +108,8 @@ func (*Extras[C, R]) getter() (g ExtraPayloadGetter[C, R]) { return }
 
 // mustBeStruct panics if `T` isn't a struct.
 func mustBeStruct[T any]() {
-	if k := reflect.TypeFor[T]().Kind(); k != reflect.Struct {
+	var x T
+	if k := reflect.TypeOf(x).Kind(); k != reflect.Struct {
```

Comment:
For my own understanding, why is `TypeOf` preferred to `TypeFor` here?

Reply:
`TypeFor` was only introduced in `1.22` and because we're on an old geth upstream it's on an older Go version.